docker/Dockerfile-dhvirtualenv

```diff
@@ -85,6 +85,16 @@ RUN apt-get update -qq -o Acquire::Languages=none \
         libpq-dev \
         xmlsec1
 
+# Debian 10 (Buster)'s python3-pip package includes pip 18.1. This version was unable to
+# pull wheels. (I'd guess we need support for manylinux2010, manylinux2014 or PEP 600,
+# introduced in pip 19.0, 19.3 and 20.3 respectively.)
+# In any case, we upgrade pip rather than compile CPython extensions from source.
+RUN pip3 install -U pip
```

Comment:
doesn't the `--upgrade-pip` in the `dh_virtualenv` invocation take care of this.

Reply:
Resolved in f0d72acebfd373b6428e8841898e90851a941192